src/breaking-change.ts

```diff
@@ -54,20 +92,73 @@ function blobHref(file: unknown) {
  * @param newSpec Path to the new swagger specification file.
  */
 async function runOad(oldSpec: string, newSpec: string) {
-  if (oldSpec === null || oldSpec === undefined || typeof oldSpec.valueOf() !== 'string' || !oldSpec.trim().length) {
-    throw new Error('oldSpec is a required parameter of type "string" and it cannot be an empty string.');
+  if (
+    oldSpec === null ||
+    oldSpec === undefined ||
+    typeof oldSpec.valueOf() !== "string" ||
+    !oldSpec.trim().length
+  ) {
+    throw new Error(
+      'oldSpec is a required parameter of type "string" and it cannot be an empty string.'
+    );
   }
 
-  if (newSpec === null || newSpec === undefined || typeof newSpec.valueOf() !== 'string' || !newSpec.trim().length) {
-    throw new Error('newSpec is a required parameter of type "string" and it cannot be an empty string.');
+  if (
+    newSpec === null ||
+    newSpec === undefined ||
+    typeof newSpec.valueOf() !== "string" ||
+    !newSpec.trim().length
+  ) {
+    throw new Error(
+      'newSpec is a required parameter of type "string" and it cannot be an empty string.'
+    );
   }
 
   console.log(`>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>`);
   console.log(`Old Spec: "${oldSpec}"`);
   console.log(`New Spec: "${newSpec}"`);
   console.log(`>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>`);
 
-  let result = await oad.compare(oldSpec, newSpec, { consoleLogLevel: 'warn' });
+  let result = await oad.compare(oldSpec, newSpec, { consoleLogLevel: "warn" });
+  let oadResult = JSON.parse(result) as OadMessage[];
+
+  const pipelineResultData: format.ResultMessageRecord[] = oadResult.map(
+    (it) => ({
+      type: "Result",
+      level: it.type as format.MessageLevel,
+      message: it.message,
+      code: it.code,
+      id: it.id,
+      docUrl: it.docUrl,
+      time: new Date(),
+      extra: {
+        mode: it.mode,
+      },
+      paths: [
+        {
+          tag: "New",
+          path: blobHref(
+            utils.trimGithubStyleFilePath(
+              utils.trimSwaggerPath(it.new.location || "")
```

Comment:
need to check if it.new is null

Reply:
If oad runs no error, it will make sure to return `new` and `old` property.